README.md

```diff
@@ -157,6 +157,64 @@ datastore.update({
 });
 ```
 
+###  scan
+
+Scan a record in the datastore. Returns `[]` if the record is empty. Returns error if record does not exist.
+
+**Arguments**
+
+* `config` - An `object`. Each of its properties defines your save operation
+* `config.table` - A `string`. The DynamoDB table name
+* `config.params` - An `object`. Each of its properties defines the query parameters
+* `config.paginate` - An `object`. Each of its properties defines the pagination parameters
+* `config.paginate.count` - A `number`. The number of items per page
+* `config.paginate.page` - An `number`. This is the page number of the set you wish for the datastore to return
+* `callback(err, result)`  - A callback which is called when the task is completed. It receives the `err` and `result`, where `result` is the list of records in the datastore. Returns `[]` if the record is empty. Returns error if the record does not exist.
```

Comment:
`Returns error if the record does not exist`?
What does that mean? It returns `[]` if there is nothing in the datastore right?